crates/rooch-rpc-api/src/jsonrpc_types/transaction_view.rs

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) RoochNetwork
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::jsonrpc_types::{
+    TransactionExecutionInfoView, TransactionSequenceInfoView, TransactionView,
+};
+use moveos_types::transaction::TransactionExecutionInfo;
+use rooch_types::transaction::{TransactionSequenceInfo, TypedTransaction};
+use schemars::JsonSchema;
+use serde::{Deserialize, Serialize};
+
+#[derive(Debug, Clone)]
+pub struct TransactionReturn {
```

Comment:
`TransactionReturn` is not a good name. How about `TransactionResult`? `TransactionWithInfo`?

Reply:
I merged this PR first. We can discuss the name later.